reference/price_lists.v3.yml

```diff
@@ -1395,14 +1392,10 @@ paths:
       summary: Upsert Price List Records
       description: |-
         Creates or updates *Price List Records*. 
-
         **Required Fields**
         * currency
-
         **Notes**
-
-        * Batch updates are supported by this endpoint, meaning you can update several price objects in one request. This update allows you to do the same work as many individual requests to singleton endpoints.
-        * When updating a product with variants, or multiple SKUs, don't include records for the parent product SKU.
```

Comment:
this line should for sure be out?

Reply:
@megdesko are we keeping these notes? I removed them since I didn't see them in the original PR. 👁️

Reply:
I would think we still want these?  If they add records for base variants, those will fail to go through.

Reply:
Sounds good, working on it!

Reply:
Additionally: we may be able to remove the "batch updates" bit- that was here because this was the very first batch endpoint at BC :-)